composer.json

```diff
@@ -7,19 +7,23 @@
     "homepage": "https://github.com/contributte/elasticsearch",
     "authors": [
         {
-            "name": "Vojtěch Mareš",
-            "email": "vojtech.mares@fiolasoft.cz"
+            "name": "Vojtěch Mareš"
         }
     ],
     "require": {
-        "php": ">=7.1",
+        "php": "^7.2",
         "nette/di": "~2.4.12",
```

Comment:
Tests are failing with php 7.2 because of old nette/utils version. You must require at least ~2.4.8

Reply:
My bad, I accidentally mixed two PRs